src/runner.rs

```diff
@@ -224,4 +228,37 @@ When your transaction is complete, enter 'commit' or 'abort' as appropriate."#,
         }
         Ok(())
     }
+
+    pub(crate) async fn handle_status(&self) -> Result<()> {
+        // TODO: Return latency information from recent commands if we're able to capture it.
+        self.deps.ui.println(&format!("Current region: {}", self.deps.env.region().value.name()));
+        self.deps.ui.println(&format!("Current ledger: {}", self.deps.driver.ledger_name()));
+        self.deps.ui.println(&format!("Shell version: {}", env!("CARGO_PKG_VERSION")));
+        Ok(())
+    }
+
+    pub(crate) async fn handle_ping(&self) -> Result<()> {
+        let region = self.deps.env.region().value.name().to_string();
+
+        let request_url = match self.deps.env.qldb_session_endpoint().value {
+            Some(url) => url.join("ping")?,
+            None => Url::parse(&format!("https://session.qldb.{}.amazonaws.com/ping", region))?
+        };
+
+        let start = Instant::now();
+        let response = reqwest::get(request_url).await?;
```

Comment:
"reqwest" is a cool spelling, but sadly wrong.

Reply:
Thats the package name: https://docs.rs/reqwest/0.11.3/reqwest/

Reply:
lol